docs/introduction-to-smart-contracts.rst

```diff
@@ -34,8 +34,8 @@ Storage
 The first line simply tells that the source code is written for
 Solidity version 0.4.0 or anything newer that does not break functionality
 (up to, but not including, version 0.6.0). This is to ensure that the
-contract is not compilable with a new (breaking) compiler version, where it could behave differently.
-So-called pragmas are common instructions for compilers about how to treat the
+contract is not compatible with a new (breaking) compiler version, where it could behave differently.
```

Comment:
This was not a typo.

Reply:
@chriseth "so-called" really isn't used very much in English and sounds kind of strange, last time we discussed this I spoke with a lot of other writers whom all agreed, so I have been slowly removing it as I come across it as part of other edits. It makes the text read like you're not very confident that this is what the term is, and leaves people wondering quite what is meant.

Pragma is also a pre-existing term, so Solidity hasn't even invented it to warrant marking it as a "made up word". If you want, then really just surrounding its introduction here in quotes is perfectly fine and much more normal.

Reply:
The comment is in line 37 - I was talking about "compilable".

Reply:
@chriseth Ah ha! 😊Reverted.